File: DTOs/Sources/PosterServiceDTOs/PosterResponseDTO.swift
import Foundation

public struct PosterResponseDTO: Codable {
    public var id: UUID
    public var name: String
    public var description: String?
   
    public init(id: UUID, name: String, description: String? = nil) {
        self.id = id
        self.name = name
        self.description = description
    }
}

File: android/dtos/PosterServiceDTOs/PosterResponseDTO.kt
package net.ipv64.kivop.dtos

//
//  PosterResponseDTO.swift
//  poster-service
//
//  Created by Dennis Sept on 26.11.24.
//
data class PosterResponseDTO (
    var id : UUID?,
    var name : String,
    var description : String?,
    var imageBase64 : String, // Hinzugefügt
)
